.github/workflows/docs.yaml

```diff
@@ -0,0 +1,67 @@
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "!**-rc"
+    paths:
+      - docs/**
+      - .github/workflows/docs.yml
```

Comment:
Do we need this?
I think that this workflow isn't ran when a tagged commit doesn't change `docs/**`.

Reply:
I don't follow you.

I've added these two paths so that this workflow is executed only if either the workflow itself is changed or any file in `docs/` folder.
There is no need to build and deploy the docs for Rust/Python code changes.

Reply:
Could you confirm whether the following case works on your fork?

1. `echo test >> README.md`
2. `git add README.md` (`docs/**` nor `.github/workflows/docs.yaml` aren't changed)
3. `git commit -m 'test'`
4. `git tag -a -m "test" 0.8.0` (This may not trigger this workflow because the last commit doesn't change `docs/**` nor `.github/workflows/docs.yaml`.)
5. `git push`

Reply:
I think this is how it should work! If there are no changes in the docs/ between two versions then there is no need to upload new HTMLs.
I guess there will be at least change of the version in the docs, e.g. 0.7.0 -> 0.8.0, so it will deploy the site.

But I am fine to remove the `paths` and let it always deploy.

Reply:
Removed `paths`!

Reply:
> Removed `paths`!

OK! We can improve this (skip needless deployments) as a follow-up task if it's needed. 

> I guess there will be at least change of the version in the docs, e.g. 0.7.0 -> 0.8.0, so it will deploy the site.

I'm not sure that GitHub Actions use `0.7.0..0.8.0` for `0.8.0` tag...